docs/config.adoc

```diff
@@ -474,24 +479,28 @@ to intercept them all. However, you may specify exact keyboard devices from the
 )
 ----
 
-If you want to specify multiple keyboards, you can separate the paths with a
-colon `+:+`.
+If a path contains space characters you should surround it with `"` characters.
+Otherwise it's optional to use them.
 
 .Example:
 [source]
 ----
 (defcfg
-  linux-dev /dev/input/dev1:/dev/input/dev2
+  linux-dev "/dev/input/input device 1"
```

Comment:
actually, single a devices path must be surrounded by () too, because otherwise it will be parsed wrong if it contains `:`